packages/manager/apps/pci-billing/src/components/consumption/ColdArchiveList.component.spec.tsx

```diff
@@ -0,0 +1,48 @@
+import { render, screen } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import ColdArchiveList from './ColdArchiveList.component';
+
+describe('ColdArchiveList', () => {
+  const mockColdArchives = [
+    {
+      name: 'archive',
+      quantity: {
+        value: 0,
+        unit: 'Hour',
+      },
+      totalPrice: 0,
+      region: 'RBX-ARCHIVE',
+    },
+    {
+      name: 'restore',
+      quantity: {
+        value: 0,
+        unit: 'Unit',
+      },
+      totalPrice: 0,
+      region: 'RBX-ARCHIVE',
+    },
+    {
+      name: 'archive-fees',
+      quantity: {
+        value: 0,
+        unit: 'Hour',
+      },
+      totalPrice: 0,
+      region: 'RBX-ARCHIVE',
+    },
+  ];
+
+  it('renders the datagrid when coldArchives are present', () => {
+    render(<ColdArchiveList coldArchives={mockColdArchives} />);
+
+    const datagrid = screen.getByTestId('datagrid');
```

Comment:
please use snapshots as it will provide a more reliable test than just testing that the datagrid is visible in the dom (at least it will provide some comparison with the data in input and the data displayed on screen)